examples/templates/action-feedback.html

```diff
@@ -0,0 +1,303 @@
+---
+layout: default
+title: Action feedback
+category: _templates
+---
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-3">
+      <h3>Base</h3>
+      <h4 class="p-muted-heading">Indeterminate</h4>
+      <button class="p-button--base p-action-feedback is-indeterminate is-cancellable js-indeterminate">
+        <span class="p-action-feedback__text">Loading...</span>
+      </button>
+      <h4 class="p-muted-heading">Determinate</h4>
+      <button class="p-button--base p-action-feedback is-determinate is-cancellable js-determinate">
+        <svg class="p-action-feedback__svg">
+          <circle class="p-action-feedback__circle" r="7" cx="8" cy="8"></circle>
+        </svg>
+        <span class="p-action-feedback__text">75%</span>
+      </button>
+      <h4 class="p-muted-heading">Determinate steps</h4>
+      <button class="p-button--base p-action-feedback is-stepped is-cancellable js-stepped">
+        <svg class="p-action-feedback__svg">
+          <circle class="p-action-feedback__circle" r="7" cx="8" cy="8"></circle>
+        </svg>
+        <span class="p-action-feedback__text">2 of 5</span>
+      </button>
+      <h4 class="p-muted-heading">Non-cancellable</h4>
+      <button class="p-button--base p-action-feedback js-noncancel">
+        <span class="p-action-feedback__text">Action</span>
+      </button>
+      <h4 class="p-muted-heading">Done</h4>
+      <button class="p-button--base p-action-feedback is-done js-indeterminate">
+        <span class="p-action-feedback__text">Done</span>
+      </button>
```

Comment:
Changing the button label to “Done” has the same drawbacks as changing it to progress text: it alters the button width, and causes adjacent elements to jump horizontally. Instead, I specified that the label should be _replaced by_ a checkmark for a couple of seconds until the dialog closes.